data/blog/8-week-cadence.md

```diff
@@ -0,0 +1,86 @@
+
+---
+
+In 2019, Electron moved to a 12 week release cycle to match Chromium's 6 week release cycle. Recently, both Chrome and Microsoft announced changes that made us re-consider Electron's current release cadence:
+
+1. Chromium plans to [release a new milestone every **4 weeks**, starting with Chrome 94 on September 21st, 2021.](https://developer.chrome.com/blog/faster-release-cycle/) This release cadence also adds a new Extended Stable option every 8 weeks, which will contain all updated security fixes.
+
+2. The Microsoft Store will [require Chromium-based apps to be no older than within 2 major versions](https://docs.microsoft.com/en-us/windows/uwp/publish/store-policies#102-security) (for example, if the latest released major version of Chromium is 85, any browser based on Chromium must be on at least Chromium version 83 or higher). This rule includes Electron apps.
+
+**Beginning in 2022, Electron will release a new major stable version every 8 weeks**, to match Chromium's 8 week Extended Stable releases.
+
+Our first release with Chromium Extended Stable will be **Electron 15** on **September 21st, 2021**.
+
+However, this change will also impact the remainder of our 2021 releases, and we wanted to let our developer community know as soon as possible. Read on for more details about our 2021 release schedule.
+
+## Initial Plan for Releases
+
+Given that our original Electron 15 release targeted a non-Extended Stable version (Chromium's Extended Stable versions are based on their even-numbered versions), but Electron apps must use the most recent 2 major versions of Chromium in their apps, our Releases WG faced a timing dilemma. 
+
+Below was the initial planned release schedule for 2021:
+
+| Electron | Chrome | Release | Beta to Stable (Weeks) |
+| -------- | ------ | ------- | ----- |
+| E13 | M91 | 2021-May-25 |  12  |
+| E14 | M93 | 2021-Aug-31 |  14  |
+| E15 | M94 | 2021-Sep-21 |  3  |
+| E16 | M96 | 2021-Nov-16 |  8  |
+| E17 | M98 | 2022-Feb-01 |  11  |
+
+Moving Electron 15 to include Chromium M94 would allow app developers to get on the very first Extended Stable version of Chromium - however, it would also shorten the beta-to-stable cycle to only 3 weeks. Yikes!
+
+We've received feedback from the Electron community in the past that our release cycle was already aggressive, and we didn't want to put Electron developers in a position where they felt rushed to update your app.
+
+## Enter the Alpha Channel
+
+To help with this switchover, Electron will begin offering an **alpha build**, one that will contain [the current Chromium Dev build version](https://chromiumdash.appspot.com/releases?platform=Mac). The alpha build will allow developers to test drive changes earlier, without needing to download more unstable nightlies. For the switchover to Chromium's new cadence, adding an alpha build would allow developers more time to test and plan for a shortened upgrade, with a more stable build.
```

Comment:
This is tricky phrasing, because this will currently only impact v15, and technically speaking our betas already cover dev versions of Chromium.  The only reason we need alpha here is the overlap

Reply:
Ahhh, so I might have misunderstood here - we're only doing a single alpha for the 15 release, not long-term?

Reply:
I want to do it longer term, but we don't have consensus on that as an idea and we need to prioritize 15 rather than this Nice To Have future.

So yeah, currently the alpha concept is a 15 one-shot

Reply:
Thanks Sam. Do you think we should communicate that E15 is sort a test run for future alpha potential? Or, too complicated?

Reply:
IMO, but I think I'm going to draft this initially as just a one shot so we don't potentially tease something that we end up not doing; depending on input from the rest of the group, we can modify that 🙂